src/components/PostCard/TopicCard.tsx

```diff
@@ -1,7 +1,11 @@
 import { useNavigation } from "@react-navigation/native";
 import { Icon } from "@rneui/themed";
 import { StackScreenPropsGeneric } from "@src/@types/navigation";
+import { getTextIpfs } from "@src/services/viewService/getContent";
```

Comment:
So you forget to change this API, do you?

Reply:
yes,i fix it